docs/Bitbucket.md

```diff
@@ -0,0 +1,169 @@
+
+## Export (`export`)
+
+| Key | Supported? | Notes |
+| --- | --- | --- |
+| `export` | ❌ | Bitbucket Premium option of sharing pipeline configuration between workspaces. Not applicable within Buildkite as an attribute - like functionality exists within [Pipeline Templates](https://buildkite.com/docs/pipelines/templates). |
+
+## Image (`image`)
+
+| Key | Supported? | Notes |
+| --- | --- | --- |
+| `image` | ✅ | The container image that is to be applied for each step within a Bitbucket pipeline, utilising the specified image within the [docker-buildkite-plugin](https://github.com/buildkite-plugins/docker-buildkite-plugin). This has lower precedence over per-step `image` configuration (see `pipelines.default.step.image`). <br/><br/> The `aws`, `aws.oidc`, `name`, `username` and `password` sub-properties are supported - refer to the Bitbucket pipelines [documentation](https://support.atlassian.com/bitbucket-cloud/docs/docker-image-options/) for more information. |
+
+## Options (`options`)
+
+| Key | Supported? | Notes |
+| --- | --- | --- |
+| `options` | ⚠️ | Customised options utilised throughout a Bitbucket pipeline. The `max-time` and `size` sub-properties are supported for translation within Buildkite Migration tool through to the generated Buildkite command step's `timeout_in_minutes` and agent tag respectfully. <br/></br> The `docker` sub-property is not supported: and will depend on the agent configuration that the corresponding Buildkite command step is being targeted to run said job has available. <br/><br/> Both supported properties at Bitbucket pipeline [step-level definition](#pipeline-properties-pipelinestypeproperty) will have higher precedences than the two values set at `options` level. |
+
+## Pipeline Starting Conditions (`pipelines.<start-condition>`)
+
+> [!NOTE]  
+> Bitbucket Pipelines allows the configuration of various pipeline start conditions: each supporting different configuration and permissible properties:
+> - `branches`: Defines the branch-specific configuration of a Bitbucket pipeline.
+> - `custom`: A customised starting condition whereby only manual triggering is allowed.
+> - `default`: The default starting configuration of a Bitbucket pipeline if it does not fall into one of the other conditions.
+> - `pull_request`: Defines the pull-request specific configuration of a Bitbucket pipeline.
+> - `tags`: Defines the tag specific configuration of a Bitbucket pipeline.
+> 
+> For information on each of these individual starting conditions, refer to the reference within the Bitbucket Pipelines [documentation](https://support.atlassian.com/bitbucket-cloud/docs/pipeline-start-conditions/#Custom--manual--pipeline-variables).
```

Comment:
Instead of referencing the BitBucket documentation I would change this to suggestions about better practices using BuildKite (like dynamic pipelines or creating different pipelines with their corresponding trigger options)

Reply:
Added a section suggesting these best practices additionally to the translated configuration the CLI gives here 👍